projects/ampleswap/index.js

```diff
@@ -12,5 +12,5 @@ module.exports = {
     staking: staking(MasterChefContract, AMPLE),
   },
   alv: { tvl: getUniTVL({ factory: '0x01dC97C89DF7d3C616a696dD53F600aB3FF12983', useDefaultCoreAssets: true }), },
-  dsc: { tvl: getUniTVL({ factory: '0x232Ba9f3B3643ab28d28ED7ee18600708D60E5fe', useDefaultCoreAssets: true, }), },
```

Comment:
hmmm, sometimes these changes would break the server because it would be expecting chain adapter to be present. Bigger issue is, now we will lose all tvl history of dsc for this protocol, this is the reason, we set `tvl: () => ({})` in some cases (if we dont mark it as dead, since we no longer that adapter we dont care if rpc is down/adapter is broken)

Reply:
Oh yes, sorry, I forgot that detail. I will fix the 2 PRs.